### PR TITLE
Status presets SelectionDialog with storing recent

### PR DESCRIPTION
**Pull request by CODeRUS**

### CODE REVIEW

File: src/client/UI/main.qml [hunk 1]
```diff
@@ -240,6 +240,7 @@ WAStackWindow {
     ListModel {
         id: browserModel
```
Comment (tgalal) [inline src/client/UI/main.qml:241]: Where is that browserModel used? I can't see it used anywhere

Comment (CODeRUS) [inline src/client/UI/main.qml:241]: don't ask me, its unchanged line from your sources =)

Comment (tgalal) [inline src/client/UI/main.qml:241]: lol you are right, sorry :)

**Comment by tgalal** [comment]
Storing all statuses in one field, separated by [WZPPRST]? This is ugly! How about we make use of the database "tables" feature we have? You should use a separate table for storing statuses, or else I can't merge it this way :(

**Pull request closed without merging**